plugins/netskope/plugin.spec.yaml

```diff
@@ -424,7 +424,7 @@ actions:
           - ""
           - exact
           - regex
-        required: false
+        required: true
```

Comment:
Why do we need to change that input parameter to `required = true`? Actually, when `type` has been not entered, only name can be updated by that action. `type` and `urls` fields should be filled out both to change URLs. If we want to change name only, we have to put a value for `name` input only. I think the problem exists here mainly because of the platform's UI - I mean the way it works.

In that situation it has to be described that if we want to update type or URLs - both fields need to be filled there (my bad that I didn't described that in help.md) 😞

Reply:
My theory was that `type` being absent negatively affects an update to the URL list, while it being `required = true` did not affect an update to the URL list name. This way a user couldn't accidentally leave it blank and get a false success which should reduce error rates.

I think either way makes sense though.

Reply:
Oh, you are right I think. It won't be able to affect name that way 😃  in case it's required to put both `type` and `urls` in code. Sounds great! 😃  In order to have that working properly -I think it's needed to remove a blank value from enum to that `type` input, and set a `default` to that for example to - `exact`.

Reply:
After discussing with yourself and looking into this more, the `type` / `urls` update is bi-directional. If `type` is not provided when updating `urls` then the update will fail. If `urls` is not provided when updating `type` then the update will fail. 

Making `type` required won't solve both problems, but making both `type` and `urls` required will. The problem with this is that neither values are required at all when updating `name`. 

I think the best solution is yours, to update the help.md with instructions for the user.